src/face.rs

```diff
@@ -41,34 +42,87 @@ pub struct Face<'a> {
     pub(crate) morx: Option<morx::Chains<'a>>,
 }
 
+impl<'a> AsRef<ttf_parser::Face<'a>> for Face<'a> {
+    #[inline]
+    fn as_ref(&self) -> &ttf_parser::Face<'a> {
+        &self.ttfp_face
+    }
+}
+
+impl<'a> AsMut<ttf_parser::Face<'a>> for Face<'a> {
+    #[inline]
+    fn as_mut(&mut self) -> &mut ttf_parser::Face<'a> {
+        &mut self.ttfp_face
+    }
+}
+
+impl<'a> core::ops::Deref for Face<'a> {
```

Comment:
Shouldn't it return `ttf_parser::Face`?

Reply:
No. This duplicates the functionality of `ttf_parser::Face`. This allows the ascender/descender/etc to be retrieved directly.

Reply:
A user should work with `ttf_parser::Face`, not `ttf_parser::FaceTables`. Or you simply doesn't want a double-deref?

Reply:
Hmm, I thought it wouldn't generate documentation, but it turns out it does!